src/main/scala/zio/prelude/Ord.scala

```diff
@@ -283,9 +283,9 @@ object Ord extends Lawful[Ord] {
 
         @tailrec
         def loop(i: Int): Ordering =
-          if (i == j && i == k) Ordering.Equals
```

Comment:
I don't think this makes sense. We're comparing integer indices in a tight loop. We aren't interested in supporting custom notions of equality we just just want to compare these two indices. This seems to just add unnecessary overhead.

Reply:
yep, performance is worth `==`